DUI3-DX/DUI3/Speckle.Connectors.DUI/Bindings/ConfigBinding.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Runtime.Serialization;
+using Speckle.Connectors.DUI.Bridge;
+using Speckle.Core.Transports;
+using Speckle.Newtonsoft.Json;
+
+namespace Speckle.Connectors.DUI.Bindings;
+
+/// <summary>
+/// POC: Simple config binding, as it was driving Dim nuts he couldn't swap to a dark theme.
+/// How does it store configs? In a sqlite db called 'DUI3Config', we create a row for each host application:
+/// [ hash,     contents         ]
+/// ['Rhino',   serialised config]
+/// ['Revit',   serialised config]
+/// </summary>
```

Comment:
I'm not sure this is storing as it should, mine always starts in dark mode even after switching it.